Makefile

```diff
@@ -19,9 +19,10 @@ pyang/xpath_parsetab.py: pyang/xpath_parser.py
 	python -m pyang.xpath_parser
 
 test: lint
-	(cd test; $(MAKE) test)
+	which xsltproc || (echo "Please run pip install -r dev-requirements.txt";exit 1)
+	(source env.sh; $(MAKE) -C test test)
```

Comment:
`$(MAKE) -C test` is fine, but we should not source env.sh here.  The tests must work w/o that.

Reply:
The tests do *not* work without that.  They often either don't run at all, with PYANG='', or they run a system installed pyang, which is not the same version and lead to confusion.